server/package.json

```
@@ -10,6 +10,8 @@
     "start": "node dist/index.js",
     "build": "tsc",
     "dev": "ts-node-dev src/index.ts",
+    "prettier": "node node_modules/prettier/bin/prettier.cjs  --config .prettierrc 'src/**/*.ts' --write",
```

```suggestion
    "prettier": "node node_modules/prettier/bin/prettier.cjs --config .prettierrc src/**/*.ts --write",
```

I had a pathing issue on my end, this resolved it